pkg/manifests/types.go

```diff
@@ -780,6 +780,10 @@ type AdditionalAlertmanagerConfig struct {
 	Timeout *string `json:"timeout,omitempty"`
 	// Defines the TLS settings to use for Alertmanager connections.
 	TLSConfig TLSConfig `json:"tlsConfig,omitempty"`
+	// Defines proxy_url settings for Alertmanager
+	ProxyURL string `json:"proxyUrl,omitempty"`
+	// Defines no_proxy settings for Alertmanager
+	NoProxy string `json:"noProxy,omitempty"`
```

Comment:
IMHO we don't need to expose settings to the end-users and only rely on the cluster-wide proxy config.

Reply:
done